packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -93,6 +99,37 @@ const ActivityDetailsSidebar = (props: Props) => {
     )
   }
 
+  const handleShareToOrg = () => {
+    UpdateReflectTemplateScopeMutation(
+      atmosphere,
+      {scope: 'ORGANIZATION', templateId: selectedTemplate.id},
+      {onError, onCompleted}
+    )
+  }
+
+  const teamScopePopover = templateTeam && selectedTemplate.scope === 'TEAM' && (
+    <div className='w-[352px] p-4'>
+      <div>
+        This custom activity is private to the <b>{templateTeam.name}</b> team.
+      </div>
+      <br />
+      <div>
+        As a member of the team you can share this activity with the{' '}
+        <b>{templateTeam.organization.name}</b> organization so that any team can also use the
```

Comment:
+1 nit on the word "any" and below on "all", which some less technically-inclined folks may interpret to mean "all teams [in the app]".
Maybe "allow teams in the X organization..."?

Reply:
Hmm, I could go either way.

@ackernaut this is the copy from your designs, WDYT?

Reply:
What do you think about this [iteration on copy in Notion](https://www.figma.com/file/PmE9A2ryODVzFPjZFJn7qi/Parabol-Activity-Library?type=design&node-id=1381%3A32031&t=ChKuSk6LsUmoIzoT-1)

Reply:
@ackernaut that copy LGTM! I'd just change "can use it" to "can use the activity" to be specific, similar to the previous copy.